sig_tracker/about_sig/templates/about_sig/detail.html

```diff
@@ -1,12 +1,16 @@
 <img src ="{{sig.sig_logo}}">
-<h1>{{sig.sig_name}}</h1>
+<h1>
+	{{sig.sig_name}}
+</h1>
 
-<h3>{{sig.co_ordinator}}</h3>
+<h3>
+	{{sig.co_ordinator}}
```

Comment:
Please follow an indentation of 2 spaces for HTML. It is generally more preferred. Change this here and below in this file.